### ISSUE TITLE
Current configuration prevents AppSync Resolvers of type 'UNIT' using 'APPSYNC_JS' resolvers

### ISSUE THREAD

**Issue by jagoodhand** [terraform-aws-modules/terraform-aws-appsync#54]
### Behaviour

The current definition of this module doesn't allow for APPSYNC_JS resolvers of type UNIT. For example, something like:

```hcl
resolvers = {
    "Query.myGetQuery" = {
    code = file("resolvers/build/Query.myGetQuery.js")
    runtime = {
        name                   = "APPSYNC_JS"
        runtime_version = "1.0.0"
    }
    kind               = "UNIT"
    data_source = "my_db"
    type               = "Query"
    field               = "myGetQuery"
    }
}
```

Fails with the error:

```bash
╷
│ Error: Missing required argument
│ 
│   with module.appsync.aws_appsync_resolver.this["Query.myGetQuery"],
│   on .terraform/modules/appsync/main.tf line 186, in resource "aws_appsync_resolver" "this":
│  186: resource "aws_appsync_resolver" "this" {
│ 
│ "runtime": all of `code,runtime` must be specified
╵
```

### Expected

These kind of resolvers are allowed, and would be expected to be deployed. For example, using the following definition deploys correctly:

```hcl
resource "aws_appsync_resolver" "appsync_js_resolver" {
  api_id = module.appsync.appsync_graphql_api_id
  field    = "myGetQuery"
  type    = "Query"
  runtime {
    name            = "APPSYNC_JS"
    runtime_version = "1.0.0"
  }
  code              = file("resolvers/build/Query.myGetQuery.js")
  data_source = "my_db"
}
```

## Possible Cause

I believe that this line is likely the issue:

https://github.com/terraform-aws-modules/terraform-aws-appsync/blob/fb7debd6ec861ab6efb051cec0b48279313b05fe/main.tf#L204

It only passes the code through if `kind === 'PIPELINE'`, which isn't a requirement of the underlying terraform module.

**Comment by antonbabenko** [comment]
Good catch. Please make a pull-request with the required fix.

**Comment by antonbabenko** [comment]
This issue has been resolved in version 2.2.1 :tada:
